pkgs/top-level/all-packages.nix

```diff
@@ -23706,6 +23706,8 @@ in
 
   ib-controller = callPackage ../applications/office/ib/controller { jdk=oraclejdk8; };
 
+  vnote = callPackage ../applications/office/vnote { };
```

Comment:
From the nixpkgs manual:

> Call your application expression using libsForQt5.callPackage instead of callPackage. Import dependencies unqualified, i.e., qtbase not qt5.qtbase. Do not import a package set such as qt5 or libsForQt5.

Reply:
fixed, will push shortly